### DISCUSSION TITLE
Handling CSS and Js after Rendering with Mystmd

### DISCUSSION THREAD

**Discussion by artafather** [executablebooks/meta#1148] [Q&A]
Hii @rowanc1

I've been trying to combine multiple markdown cells and split the rendered output to get something like Jupyterbook with different cells while having shared references among cells . This seem to require modifying CSS and js code after rendering to be able to split the rendered stuff and place in their cell. But currently I could only see an HTML renderer. Is there an existing functionality that also renders the CSS and js code for more control?

Cheers,
Ashra

**Comment by agoose77** [comment]
I'm closing this in favour of executablebooks/mystmd#1004
